pavelib/servers.py

```diff
@@ -48,40 +54,63 @@ def run_server(system, settings=None, port=None, skip_assets=False, contracts=Fa
 @needs('pavelib.prereqs.install_prereqs')
 @cmdopts([
     ("settings=", "s", "Django settings"),
+    ("asset-settings=", "a", "Settings file used for updating assets. Defaults to settings if not provided."),
     ("port=", "p", "Port"),
-    ("fast", "f", "Skip updating assets")
+    ("fast", "f", "Skip updating assets"),
+    ("no-contracts", "f", "Disable contracts. By default, they're enabled in devstack."),
 ])
 def lms(options):
     """
     Run the LMS server.
     """
-    settings = getattr(options, 'settings', None)
+    settings = getattr(options, 'settings', DEFAULT_SETTINGS)
+    asset_settings = getattr(options, 'asset-settings', settings)
     port = getattr(options, 'port', None)
     fast = getattr(options, 'fast', False)
-    run_server('lms', settings=settings, port=port, skip_assets=fast)
+    collect_static = not fast and asset_settings != settings
```

Comment:
This is what I intended (I did it in a few places). My thinking was that if the asset file was different than the regular settings file, that this meant that the user was generating static files that they were going to use. This isn't a perfect definition so maybe it would be better to make it an explicit parameter, but I don't imagine that anyone will use it.